packages/player/src/components/hls-source.tsx

```diff
@@ -71,19 +71,32 @@ export const HLSSource: React.FC<HLSSourceProps> = ({
     const canUseNative = video.canPlayType('application/vnd.apple.mpegurl')
     const shouldUseNative = canUseNative && !Hls.isSupported()
 
+    function handleExitedFullscreenOnIphone() {
+      videoService.send('PAUSE')
+    }
+
     // Check for Media Source support
     if (!shouldUseNative) {
       _initPlayer()
     } else {
       video.load()
+      video.addEventListener(
```

Comment:
we want to monitor for this event on iOS specifically because the video pauses coming out of fullscreen but doesn't explicitly set an event 🥴